docs/framework/guides/contributing/development-environment.md

```diff
@@ -133,13 +133,58 @@ yarn run docs:verify
+* `--concurrency` (alias `-c`) &ndash; Number of concurrent pages (browser tabs) to be used during the scan. By default all links are opened one by one, sequentially (concurrency is 1).
+* `--quit` (alias `-q`) &ndash; A boolean argument that specifies, whether the scan should be terminated as soon as the first error is found. Disabled by default, so all found links are scanned regardless of whether they have errors or not.
+
+For example, to check the documentation without the default exclusions (the API and assets links), using only 2 concurrent pages and terminate the scan as soon as first error is found, run this command:
+
+```
+yarn run docs:verify -e -c 2 -q
+```
+
+#### Defining exclusions for web crawler
+
+The crawler supports exclusions provided as text patterns, which are then searched for in the error messages as a substrings. This pattern is just a plain text, not a regular expression. When a pattern (substring) is found, such an error is ignored - it is not listed after the finished scan and it does not mark the entire scan as failed. Only non-ignored errors are logged at the end of the scan.
+
+The pattern for ignoring an error must be defined in `<meta>` tag on a page, where an error occurs. This `<meta>` tag must have `x-cke-crawler-ignore-patterns` name and `content` value provided as JSON object. Each key in this object defines text patterns for a separate type of errors, that can be detected by the crawler. Each value is the text or an array of texts, that are used for finding a match in error messages. The special wildcard value `*` is used to ignore all errors that are found for given error type.
+
+The following error types are supported: `uncaught-exception`, `request-failure`, `response-failure`, `console-error`, `navigation-error` and `page-crash`:
+
+* `uncaught-exception` &ndash; As the name suggests, these are uncaught exceptions from the page.
+* `request-failure` &ndash; It can occur, when the request has not been sent (e.g. it was blocked by the browser) or has not received any response (e.g. due to a timeout). HTTP error responses, such as 404 or 500, are considered as successful ones from HTTP standpoint, so such requests will not be logged as request failures, but as response failures.
+* `response-failure` &ndash; Each HTTP response with status code equal or greater than 400 is treated as failed one.
+* `console-error` &ndash; All `console.error()` calls are treated as an error.
+* `navigation-error` &ndash; The navigation error may happen, when:
+	* there's an SSL error (e.g. in case of self-signed certificate or expired one),
+	* target URL is invalid,
+	* the remote server does not respond or is unreachable,
+	* the timeout is exceeded during navigation to a page, so the `load` event is not emmited (e.g. due to an infinite loop in the JavaScript code).
+* `page-crash` &ndash; The general page malfunction, that does not fit to other categories (e.g. running out of a RAM).
```

Comment:
What do you think about describing supporting errors in a table with 3 columns: "Error name", "Supported values" (String, Array, etc), "Examples"?

Reply:
I added a table with examples for each type of error. I didn't add a column with supported values, because each error type supports the same format of the pattern: string, array of strings or wildcard value `*`. Additionally, I rewrote a few pieces of text to make them easier to read, I hope.